### PR TITLE
Change output of card

### PR DESCRIPTION
**Pull request by jiratQ**
**Card output now have flowers**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -25,15 +20,14 @@
...
+bot = commands.Bot(command_prefix="$")
 
-sys.path.insert(0, os.path.abspath("src/utils/"))
 
 @bot.command(name="hello")
 async def nine_nine(ctx):
     channel = bot.get_channel(ctx.channel.id)
     message_id = channel.last_message_id
-    await ctx.send("HI :flushed:")
+    await ctx.send("HI :heart:")
```
Comment (phusitsom) [inline main.py:30]: BAD

File: main.py [hunk 2]
```diff
@@ -54,20 +48,20 @@ async def poker(ctx):
     )
 
 @bot.command(name='voice')
-async def audio_say(ctx, *,sound = None):
+async def audio_say(ctx, sound=None):
```
Comment (phusitsom) [inline main.py:51]: Please undo :flushed:

File: main.py [hunk 3]
```diff
@@ -54,20 +48,20 @@ async def poker(ctx):
...
+
 @bot.command(name='disconnect')
 async def audio_disconnect(ctx):
     await disconnect(ctx)
 
-@bot.command(name='snap')
-async def snap_kick(ctx , user: discord.Member = None):
-    await random_kick(bot, ctx, user)
-
 @bot.event
```
Comment (phusitsom) [inline main.py:62]: sad

File: main.py [hunk 4]
```diff
@@ -54,20 +48,20 @@ async def poker(ctx):
...
 async def on_message(message):
     await bot.process_commands(message)
-    await guess_command(bot, message, bot.all_commands)
+    await guess_command(bot, message)
```
Comment (phusitsom) [inline main.py:65]: Please redo

File: main.py [hunk 5]
```diff
@@ -54,20 +48,20 @@ async def poker(ctx):
...
 
 @bot.command(name='voice')
-async def audio_say(ctx, *,sound = None):
+async def audio_say(ctx, sound=None):
     await voice(bot, ctx, sound)
 
+@bot.command(name='repeat')
+async def audio_repeat(ctx,* , text):
+    await repeat(bot, ctx, text = text)
+
```
Comment (phusitsom) [inline main.py:57]: Delete this

File: main.py [hunk 6]
```diff
@@ -25,15 +20,14 @@
 
 TOKEN = os.getenv("TOKEN")
 
-bot = commands.Bot(command_prefix=config.prefix)
+bot = commands.Bot(command_prefix="$")
```
Comment (phusitsom) [inline main.py:23]: Please undo

**Review by phusitsom — CHANGES_REQUESTED** [review]
Please undo changes in main.py

File: main.py [hunk 7]
```diff
@@ -1,21 +1,16 @@
...
 
 from discord.ext import commands
 from dotenv import load_dotenv
 
 from src.utils.party import get_players
 from src.utils.utils import guess_command
 from src.utils.kick import random_kick
-
-from src.utils.party import get_players
-from src.utils.utils import guess_command
```
Comment (namirinz) [inline main.py:12]: Why?

File: main.py [hunk 8]
```diff
@@ -25,15 +20,14 @@
 
 TOKEN = os.getenv("TOKEN")
 
-bot = commands.Bot(command_prefix=config.prefix)
+bot = commands.Bot(command_prefix="$")
 
-sys.path.insert(0, os.path.abspath("src/utils/"))
```
Comment (namirinz) [inline main.py:30]: Why?

**Review by namirinz — CHANGES_REQUESTED** [review]

File: main.py (hunk 7 above)
Comment (jiratQ) [reply main.py:12]: already have

File: main.py (hunk 8 above)
Comment (jiratQ) [reply main.py]: **sorry I will fix it**

File: main.py [hunk 9]
```diff
@@ -50,7 +51,7 @@ async def poker(ctx):
 
 @bot.command(name='voice')
 async def audio_say(ctx, *, sound=None):
-    await voice(ctx, sound=None)
+    await voice(bot, ctx, sound)
```
Comment (phusitsom) [inline main.py]: bad

**Review by phusitsom — CHANGES_REQUESTED** [review]

File: src/poker/poker.py [hunk 10]
```diff
@@ -53,10 +53,10 @@ async def show_middle_card(middle_cards: List[int], ctx, show_four: bool, show_f
 
     if show_four:
         four_card_msg = DECKS_OF_CARDS[middle_cards[3]]
-        msg += '  ' + four_card_msg
+        msg += '   ' + four_card_msg
     if show_five:
         five_card_msg = DECKS_OF_CARDS[middle_cards[4]]
-        msg += '  ' + five_card_msg
+        msg += '   ' + five_card_msg
```
Comment (namirinz) [inline src/poker/poker.py:59]: Nice try

File: src/poker/poker.py [hunk 11]
```diff
@@ -80,13 +96,16 @@ async def who_win(middle_card: List[int], ctx, player_cards: List[Tuple[int, int
...
     for i in range(len(win)):
-        msg += str(win[i][3]) + ' โดยถือไพ่ ' + \
-            str(win[i][2]) + ' คือระดับ ' + win[i][1] + ','
-
+        card=' '
+        msg += str(win[i][4]) + ' โดยถือไพ่\n' 
+        for j in range(len(win[i][2])):
+            card += str(dic[win[i][2][j]])+' '+str(win[i][3][j]) + '   '
+        msg += card
+        msg +=  '\nคือระดับ ' + win[i][1] + ','
```
Comment (namirinz) [inline src/poker/poker.py:106]: Well  Done!

**Review by namirinz — APPROVED** [review]
Approved!

File: src/poker/poker.py (hunk 10 above)
Comment (phusitsom) [reply src/poker/poker.py:59]: ok

Comment (jiratQ) [reply src/poker/poker.py:59]: Thank You

File: src/poker/poker.py (hunk 11 above)
Comment (jiratQ) [reply src/poker/poker.py:106]: THAK U BRO

**Review by phusitsom — APPROVED** [review]

**Pull request merged**